lisp/ess-r-mode.el

```diff
@@ -2293,7 +2293,8 @@ state.")
             (end (match-end 0))
             ;; Remove surrounding quotes so the text property is the
             ;; name of the link:
-            (text (string-trim (match-string 0) "[\"‘]*" "[\"’]*")))
+            (text (replace-regexp-in-string "^[\"']+" "" 
```

Comment:
I've already fixed this in master. Please don't include changes that aren't related to the PR, it makes it harder to review.